app/scripts/services/aggregatedLoggingService.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .factory("AggregatedLoggingService", function($q, $cacheFactory, Logger, $interval, APIService, DataService) {
+
+    var cachedUserPermissions = $cacheFactory('operationsUsersCache', {
+      number: 10
+    });
+
+    var inFlightPermissionsRequest = null;
+    var authPromise = null;
+
+    // Create SelfSubjectAccessReview request againt authorization API to check whether
+    // current user can view pods/log from 'openshift-logging' project.
+    // Users who can view such logs are 'operations' users
+    var isOperationsUser = function(userName) {
```

Comment:
This should not need to take in a user name since it uses self subject access review. (The cached data will be cleared on logout because the page is reloaded, so that's not a problem here.)